README.dev-charms.md

```diff
@@ -87,14 +87,16 @@ Read the OpenStack [development documentation][] for full details.
 
 ## Stable charm updates
 
-Any update to a stable charm must first be applied into the master branch; it should then be cherry-picked in a review for the stable branch:
+Any update to a stable charm must first be applied into the master branch; it should then be cherry-picked in a review for the stable branch corresponding your target release (ensuring that any interim releases have the fix landed):
 
 ```
-git checkout -b bug/XXXXX-stable stable
+git checkout -b bug/XXXXX-stable stable/YY.MM
```

Comment:
interestingly I generally name my branches:

stable/bug/XXXXX

which seems to work OK as well.